core/src/main/java/com/datastax/oss/driver/api/core/data/CqlVector.java

```diff
@@ -196,7 +233,8 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return Iterables.toString(this.list);
+    TypeCodec<T> subcodec = CodecRegistry.DEFAULT.codecFor(list.get(0));
```

Comment:
Previously we promised that `CqlVector.from` should mirror `CqlVector.toString`. `CqlVector.from` has access to a subtype codec and expects some values to be single-quoted, e.g. texts. Therefore, without this change to `toString`, a round-trip test like `CqlVector.from(vector1.toString(), codec);` will fail. However, fetching the default codec for the subtype looks like too much effort for something simple like `toString`.